src/math/p5.Vector.js

```diff
@@ -2042,19 +2042,19 @@ p5.Vector.rem = function rem(v1, v2) {
  * @static
  * @param  {p5.Vector} v1 a <a href="#/p5.Vector">p5.Vector</a> to subtract from
  * @param  {p5.Vector} v2 a <a href="#/p5.Vector">p5.Vector</a> to subtract
- * @param  {p5.Vector} target if undefined a new vector will be created
- */
-/**
- * @method sub
- * @static
- * @param  {p5.Vector} v1
- * @param  {p5.Vector} v2
+ * @param  {p5.Vector} [target] if undefined a new vector will be created
```

Comment:
New vector won't be created if passed `undefined` so this description needs to change as well.

Reply:
am still confused about what appropriate description should be for this target parameter....like if target(p5.Vector) is passed it's value will be set to result of function and if it's not passed or undefined it will still return resultant vector..

Any Suggestions about what should one-liner description should be🤔

Reply:
The one for `add` just above is fine. If the function signature says it will return then there's no need to say it again here.